source/docs/software/commandbased/commands-v3/mechanisms.rst

```diff
@@ -0,0 +1,434 @@
+# Mechanisms
+
+Mechanisms (called "Subsystems" in Commands v2) represent independently-controlled groupings of robot hardware. They serve two key purposes:
+
+1. **Encapsulation**: Hide hardware implementation details from the rest of your code
+2. **Resource Management**: Ensure only one command uses a mechanism at a time
+
+.. note::
+   If you're familiar with Commands v2 Subsystems, Mechanisms work almost identically. The main differences are the name change and a few API updates for v3's coroutine-based commands.
```

Comment:
It might also be worth talking in this section about what a mechanism should and shouldn't be, and why the name changed in v3. For example, https://github.com/wpilibsuite/allwpilib/pull/6518#issuecomment-3198999726